### PR TITLE
docs: Remove docusaurus headers

### PR DESCRIPTION
**Pull request by javieri-empathy**
EX-5081

Removes all docusaurus related code from the build.

### CODE REVIEW

**Review by herrardo — COMMENTED** [review]
Just one comment:

In some of their static docs, they are including a header with title and subtitle...

https://raw.githubusercontent.com/empathyco/docs-empathy-documentation/main/functional/about-empathy-platform.md?token=ABDSUWP3AJVDYRLYPP4KBM3BVXNUI

So, should we remove the whole header?

**Comment by tiborux** [comment]
waiting for @rgutialo to check if we need to remove all the headers.

**Review by pmareke — APPROVED** [review]

File: packages/x-components/build/docgen/templates/docgen-component.template.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+function createComponentTemplate(renderedUsage, doc) {
+  const { displayName, description, docsBlocks } = doc;
+  const title = kebabToPascalCase(displayName);
+  return `
+  ---
+  title: ${title}
+  ---
+  # ${title}
+
+  ${description || ''}
```
Comment (herrardo) [inline packages/x-components/build/docgen/templates/docgen-component.template.js:10]: So I understand that you are overwritting the default template for the component and with this line we avoid the `>` quotation formatting

Comment (herrardo) [reply packages/x-components/build/docgen/templates/docgen-component.template.js:10]: I think we are also fixing the third point of [this task](https://searchbroker.atlassian.net/browse/EX-5063)

**Review by herrardo — APPROVED** [review]

File: packages/x-components/build/docgen/templates/docgen-component.template.js (hunk 1 above)
Comment (javieri-empathy) [reply packages/x-components/build/docgen/templates/docgen-component.template.js:10]: Accidentally, but yes, we should be solving those issues.

**Review by pmareke — APPROVED** [review]

**Pull request merged**